api.go

```diff
@@ -29,9 +29,35 @@ type DockerInfo struct {
 	ENV_ID     string `json:"environment_id" binding:"required"`
 }
 
+type config struct {
+	BUILDLOCATION  string
+	FRANKLINAPIURL string
+	ENV            string
+}
+
+var Config config
+
+// Investigate a better way to do this
+func init() {
+	Config.BUILDLOCATION = os.Getenv("BUILD_LOCATION")
+	if Config.BUILDLOCATION == "" {
+		logging.LogToFile("Missing environment variable BUILD_LOCATION")
+		panic("Missing environment variable BUILD_LOCATION")
+
+	}
+	Config.FRANKLINAPIURL = os.Getenv("API_URL")
+	if Config.FRANKLINAPIURL == "" {
+		logging.LogToFile("Missing environment variable API_URL")
+		panic("Missing environment variable API_URL")
```

Comment:
I don't love having to explicitly write these out, but I couldn't think of a cleaner way to make sure that these are required and give you and throw an error early enough.